code/datums/movement_controller/movement_controller.dm

```diff
@@ -12,4 +12,4 @@
 			SHOULD_CALL_PARENT(TRUE)
 			// stub
 
-		update_owner_dir(var/atom/movable/owner)
+		update_owner_dir()
```

Comment:
nope....

Reply:
The child datum had this proc with the "extraneous argument"; this proc is only used for the tank and pod, neither of which actually use the argument. Do you think the parent datum should keep the proc argument and the child proc should remove it?

Reply:
fair, if it's not being used anywhere and there's only these two subtypes which do it different, it's ok to remove